.github/workflows/build.yml

```
@@ -0,0 +1,51 @@
+name: Build and Push Docker Image
+
+on:
+  workflow_run:
```

actionlint (1.7.7)

no workflow is configured for "workflow_run" event

(events)